upgrade-notes/backend_20241009_135052.md

```diff
@@ -0,0 +1,4 @@
+#### Parameter groups refactor + editable in Admin ([#3484](https://github.com/shopsys/shopsys/pull/3484))
+
+-   removal of the deprecated visible attribute
+-   see #project-base-diff to update your project
```

Comment:
according to upgrade notes. we're in the middle of thinking how the upgrade notes should be written... 

currently, I think we should: 
- heading should be in form `do this`
- mention critical changes in logic (this does not apply to this PR) 
-  mention movements of features from project-base to packages - see https://github.com/shopsys/shopsys/blob/bdc02ab7cb325942af13479d23bcd80e9c976598/upgrade-notes/backend_20241007_081817.md for inspiration
- mention changes in FE API

so, according to these claims, I suggest something like 

```
#### make parameter groups editable in admin ([#3484](https://github.com/shopsys/shopsys/pull/3484))

- 
- [features moved](#movement-of-features-from-project-base-to-packages) from project-base to the framework package:
    - Parameter groups
- field `visible` was removed from GraphQL field `Parameter`
-   see #project-base-diff to update your project
```

or something like that